wled00/FX_2Dfcn.cpp

```diff
@@ -562,40 +562,73 @@ void Segment::drawLine(uint16_t x0, uint16_t y0, uint16_t x1, uint16_t y1, uint3
 #include "src/font/console_font_5x12.h"
 #include "src/font/console_font_6x8.h"
 #include "src/font/console_font_7x9.h"
+#include "src/font/console_font_12x16.h"
+#include "src/font/console_font_12x24.h"
+#include "src/font/console_font_16x32.h"
+#include "src/font/console_font_25x57.h"
 
 // draws a raster font character on canvas
-// only supports: 4x6=24, 5x8=40, 5x12=60, 6x8=48 and 7x9=63 fonts ATM
+// only supports: 4x6=24, 5x8=40, 5x12=60, 6x8=48, 7x9=63, 12x16=192, 16x24=288, 16x32=512 and 25x57=1425 fonts ATM
 void Segment::drawCharacter(unsigned char chr, int16_t x, int16_t y, uint8_t w, uint8_t h, uint32_t color, uint32_t col2) {
-  if (!isActive()) return; // not active
```

Comment:
Even though we added measures elsewhere this should remain here to prevent crash if segment changes during writing to it.